sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 #### Bugs Fixed
 
 #### Other Changes
+* Fixed shading instructions to correct dependency issues in Azure Synapse with version 4.14.0 and 4.14.1. - See [PR 31980](https://github.com/Azure/azure-sdk-for-java/pull/31980) 
```

Comment:
does this issue only happen > 4.14.0? is it a version upgrade result?

should the changelog also be added in 3-3?

Reply:
I don't think this change is relevant for Spark 3-3 - we never published Spark 3-3 to Maven with broken shading instructions?

Reply:
4.13.1 still worked fine (used that with the internal customer hitting the dependency errors when I asked them to upgrade Spark connector to latest version)

Reply:
gotcha, thanks for the context :)